sway/commands/titlebar_separator.c

```diff
@@ -0,0 +1,19 @@
+#include <string.h>
+#include <strings.h>
+#include "sway/commands.h"
+#include "util.h"
+
+struct cmd_results *cmd_titlebar_separator(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "titlebar_separator", EXPECTED_EQUAL_TO, 1))) {
+		return error;
+	} else if(strcmp(argv[0], "enable") == 0) {
+		config->titlebar_separator = false;
+	} else if(strcmp(argv[0], "disable") == 0) {
+		config->titlebar_separator = true;
```

Comment:
I believe this should be reversed. Setting `titlebar_separator enable` now disables of the separator, which might not make sense to the user. I had it set `disable` on my first test and was wondering why the separator was still there.